packages/db/prisma/tsconfig.json

```diff
@@ -1,8 +0,0 @@
-{
-  "extends": "@repo/typescript-config/react-library.json",
-  "compilerOptions": {
```

Comment:
any reason for deleting this?

Reply:
I had added it in the previous PR by mistake, so removed it